modules/statics/src/coins.ts

```diff
@@ -1006,6 +1006,26 @@ export const coins = CoinMap.fromCoins([
     BaseUnit.TCOREUM,
     COSMOS_SIDECHAIN_FEATURES
   ),
+  account(
+    '9fa0f191-4eed-4030-864a-d14bbd98c8af',
+    'rune-native',
```

Comment:
calling this as `rune-native` because `rune` is already taken by another eth token.

Reply:
aren't eth tokens in the format of `eth:tokenName` ?

Reply:
eth tokens are always in `tokenName` format